os-neutron.te

```diff
@@ -45,6 +46,7 @@ neutron_domtrans(keepalived_t)
 
 # Bugzilla 1169859 & 1171460 & 1171458
 can_exec(neutron_t,neutron_var_lib_t)
```

Comment:
Btw. this is weird, why neutron needs to exec neutron_var_lib_t files? Maybe SELinux label for that specific file should be changed to neutron_exec_t.